.eslintrc

```diff
@@ -27,7 +27,6 @@
           "afterColon": true
         }
       ],
-      "react/require-default-props": [2, { "forbidDefaultForRequired": true }],
```

Comment:
Any reason we do not want components to have default props?

Reply:
I wasn't involved with the current setup, so I don't know the reason behind some decisions, but I observed that the`className` prop has an optional string type. In some components, the className props and type is defined but not used in that file (e.g., `src/pages/about-us.tsx`), hence having default props would just be a redundant className = null or so.

Maybe the best thing, then is to remove the props and types in files that wouldn't use need it? I didn't think of this. Do let me know, and I'll revert accordingly.

Reply:
The same issue applies to so many other props, actually.

Reply:
I guess the issue, then, is that in some files, the component is reused twice, with one requiring the props and the other not. Hence the props type cannot be optional nor required.